apps/admin-gui/src/app/shared/components/dialogs/assign-group-to-resource-dialog/assign-group-to-resource-dialog.component.html

```diff
@@ -23,6 +23,11 @@ <h1 mat-dialog-title>{{'DIALOGS.ASSIGN_GROUP_TO_RESOURCE.TITLE' | translate}}</h
     </perun-web-apps-groups-list>
   </div>
   <div mat-dialog-actions>
+    <mat-checkbox
+      matTooltip="{{'DIALOGS.ASSIGN_GROUP_TO_RESOURCE.ASYNC_OPT_DESCRIPTION' | translate}}"
+      [(ngModel)]="async"
+      class="ml-5">{{'DIALOGS.ASSIGN_GROUP_TO_RESOURCE.ASYNC_OPT' | translate}}
```

Comment:
Elements in UI should be always logically aligned and there is no reason, why this should be off. Please, remove the css class.
```suggestion
      [(ngModel)]="async"
      >{{'DIALOGS.ASSIGN_GROUP_TO_RESOURCE.ASYNC_OPT' | translate}}
```

Reply:
Thanks, removed.